c_glib/arrow-glib/table-builder.h

```diff
@@ -46,8 +46,15 @@ void garrow_record_batch_builder_set_initial_capacity(GArrowRecordBatchBuilder *
 GArrowSchema *garrow_record_batch_builder_get_schema(GArrowRecordBatchBuilder *builder);
 
 gint garrow_record_batch_builder_get_n_fields(GArrowRecordBatchBuilder *builder);
```

Comment:
Could you also create `garrow_record_batch_builder_get_n_columns()`?

Reply:
I've created `garrow_record_batch_builder_get_n_columns()`.